apps/src/code-studio/initApp/project.js

```diff
@@ -522,8 +522,9 @@ var projects = module.exports = {
    *   call to `sourceHandler.getLevelSource`.
    * @param {function} callback Function to be called after saving.
    * @param {boolean} forceNewVersion If true, explicitly create a new version.
+   * @param {boolean} preparingRemix Indicates whether this save is part of a remix.
    */
-  save(sourceAndHtml, callback, forceNewVersion) {
+  save(sourceAndHtml, callback, forceNewVersion, preparingRemix) {
```

Comment:
There is a bunch of code in `project.js` that looks like it was meant to be revisited, but that never happened. Maybe a tech debt item?